src/components/LandingHero.tsx

```diff
@@ -1,26 +1,26 @@
-import { useState } from "react"
-import { useLocation, Link } from "wouter"
-import { TypeBadge } from "./TypeBadge"
 import { Button } from "@/components/ui/button"
-import { CreateNewSnippetHero } from "./CreateNewSnippetHero"
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card"
+import { Link, useLocation } from "wouter"
+import { useGlobalStore } from "../hooks/use-global-store"
+import { CreateNewSnippetHero } from "./CreateNewSnippetHero"
+import { TypeBadge } from "./TypeBadge"
 
 export const LandingHero = () => {
-  const [visible, setVisible] = useState(true)
+  const { should_onboarding_tips_be_closed, setOnboardingTipsClosed } = useGlobalStore()
```

Comment:
use a selector with the global store- it will cause less react-rerenders (important react optimization)

Reply:
changes made